lib/protos.js

```diff
@@ -231,9 +231,10 @@ exports.reset = function() {
  * (e.g. `userId` in '<img src="https://whatever.com/{{ userId }}">').
  * @param {Function} [callback=noop] A callback, invoked when the tag finishes
  * loading.
+ * @param {Boolean} moveToBody to specify whether to load the iframe in head or body
  */
 
-exports.load = function(name, locals, callback) {
+exports.load = function(name, locals, callback, moveToBody) {
```

Comment:
```suggestion
exports.load = function(name, locals, callback, shouldMoveIframeToBody) {
```

Reply:
renaming the parameter here to make it clear it only impacts iframe, but not other types of elements supported in this function.